### ISSUE TITLE
Assembly breaking with "make: *** [birdoutput-1.fa] Error 1"

### ISSUE THREAD

**Issue by aersoares81** [bcgsc/abyss#161]
Hi,
The assembly abruptly stops with the error "make: *** [birdoutput-1.fa] Error 1" for me. Any help is welcome. :)

The command line was
```
abyss-pe mpirun=mpirun k=51 np=38 name=birdoutput lib='pea peb pec ped pee pef' mp='mpg mph mpj' pea='/drive/andre/bird/PEreads/library1.R1.fq.gz /drive/andre/bird/PEreads/library1.R2.fq.gz' peb='/drive/andre/bird/PEreads/library550bpinsert.R1trimmed.fq.gz /drive/andre/bird/PEreads/library550bpinsert.R2trimmed.fq.gz' pec='/drive/andre/bird/PEreads/library350bpinsert.R1trimmed.fq.gz /drive/andre/bird/PEreads/library350bpinsert.R2trimmed.fq.gz' ped='/drive/andre/bird/PEreads/library3.pe.R1trimmed.fq.gz /drive/andre/bird/PEreads/library3.pe.R2trimmed.fq.gz' pee='/drive/andre/bird/PEreads/library2.pe.R1trimmed.fq.gz /drive/andre/bird/PEreads/library2.pe.R2trimmed.fq.gz' pef='/drive/andre/bird/PEreads/library4.pe.R1trimmed.fq.gz /drive/andre/bird/PEreads/library4.pe.R2trimmed.fq.gz' mpg='/drive/andre/bird/MPreads/matepair3kb.RF1.fastq.gz /drive/andre/bird/MPreads/matepair3kb.RF2.fastq.gz' mph='/drive/andre/bird/MPreads/matepair5kb.RF1.fastq.gz /drive/andre/bird/MPreads/matepair5kb.RF2.fastq.gz' mpj='/drive/andre/bird/MPreads/matepair12kb.RF1.fastq.gz /drive/andre/bird/MPreads/matepair12kb.RF2.fastq.gz'
```
The full output was:
```
mpirun -np 38 ABYSS-P -k51 -q3    --coverage-hist=coverage.hist -s birdoutput-bubbles.fa  -o birdoutput-1.fa /drive/andre/bird/PEreads/library1.R1.fq.gz /
drive/andre/bird/PEreads/library1.R2.fq.gz /drive/andre/bird/PEreads/library550bpinsert.R1trimmed.fq.gz /drive/andre/bird/PEreads/library550bpinsert.R2trimmed.fq.gz /dc
ap1/andre/bird/PEreads/library350bpinsert.R1trimmed.fq.gz /drive/andre/bird/PEreads/library350bpinsert.R2trimmed.fq.gz /drive/andre/bird/PEreads/library2.pe.R1trimmed.f
q.gz /drive/andre/bird/PEreads/library2.pe.R2trimmed.fq.gz /drive/andre/bird/PEreads/library3.pe.R1trimmed.fq.gz /drive/andre/bird/PEreads/library3.pe.R
2trimmed.fq.gz /drive/andre/bird/PEreads/library4.pe.R1trimmed.fq.gz /drive/andre/bird/PEreads/library4.pe.R2trimmed.fq.gz 
ABySS 2.0.2
ABYSS-P -k51 -q3 --coverage-hist=coverage.hist -s birdoutput-bubbles.fa -o birdoutput-1.fa /drive/andre/bird/PEreads/library1.R1.fq.gz /drive/andre/bemtev
i/PEreads/library1.R2.fq.gz /drive/andre/bird/PEreads/library550bpinsert.R1trimmed.fq.gz /drive/andre/bird/PEreads/library550bpinsert.R2trimmed.fq.gz /drive/andre/bird/
PEreads/library350bpinsert.R1trimmed.fq.gz /drive/andre/bird/PEreads/library350bpinsert.R2trimmed.fq.gz /drive/andre/bird/PEreads/library2.pe.R1trimmed.fq.gz /drive/andre/
bird/PEreads/library2.pe.R2trimmed.fq.gz /drive/andre/bird/PEreads/library3.pe.R1trimmed.fq.gz /drive/andre/bird/PEreads/library3.pe.R2trimmed.fq.gz /dc
ap1/andre/bird/PEreads/library4.pe.R1trimmed.fq.gz /drive/andre/bird/PEreads/library4.pe.R2trimmed.fq.gz
Running on 38 processors
0: Running on host labhost1
2: Running on host labhost1
4: Running on host labhost1
6: Running on host labhost1
8: Running on host labhost1
9: Running on host labhost1
10: Running on host labhost1
11: Running on host labhost1
12: Running on host labhost1
13: Running on host labhost1
14: Running on host labhost1
15: Running on host labhost1
16: Running on host labhost1
17: Running on host labhost1
18: Running on host labhost1
19: Running on host labhost1
20: Running on host labhost1
21: Running on host labhost1
22: Running on host labhost1
23: Running on host labhost1
24: Running on host labhost1
25: Running on host labhost1
26: Running on host labhost1
27: Running on host labhost1
28: Running on host labhost1
29: Running on host labhost1
30: Running on host labhost1
31: Running on host labhost1
32: Running on host labhost1
33: Running on host labhost1
34: Running on host labhost1
35: Running on host labhost1
36: Running on host labhost1
37: Running on host labhost1
1: Running on host labhost1
3: Running on host labhost1
5: Running on host labhost1
7: Running on host labhost1
0: Reading `/drive/andre/bird/PEreads/library1.R1.fq.gz'...
7: Reading `/drive/andre/bird/PEreads/library2.pe.R2trimmed.fq.gz'...
1: Reading `/drive/andre/bird/PEreads/library1.R2.fq.gz'...
2: Reading `/drive/andre/bird/PEreads/library550bpinsert.R1trimmed.fq.gz'...
3: Reading `/drive/andre/bird/PEreads/library550bpinsert.R2trimmed.fq.gz'...
4: Reading `/drive/andre/bird/PEreads/library350bpinsert.R1trimmed.fq.gz'...
5: Reading `/drive/andre/bird/PEreads/library350bpinsert.R2trimmed.fq.gz'...
6: Reading `/drive/andre/bird/PEreads/library2.pe.R1trimmed.fq.gz'...
8: Reading `/drive/andre/bird/PEreads/library3.pe.R1trimmed.fq.gz'...
9: Reading `/drive/andre/bird/PEreads/library3.pe.R2trimmed.fq.gz'...
10: Reading `/drive/andre/bird/PEreads/library4.pe.R1trimmed.fq.gz'...
11: Reading `/drive/andre/bird/PEreads/library4.pe.R2trimmed.fq.gz'...
`/drive/andre/bird/PEreads/library2.pe.R2trimmed.fq.gz': discarded 6583271 reads shorter than 51 bases
`/drive/andre/bird/PEreads/library2.pe.R2trimmed.fq.gz': discarded 3434 reads containing non-ACGT characters
`/drive/andre/bird/PEreads/library2.pe.R1trimmed.fq.gz': discarded 6213731 reads shorter than 51 bases
`/drive/andre/bird/PEreads/library2.pe.R1trimmed.fq.gz': discarded 5816 reads containing non-ACGT characters
`/drive/andre/bird/PEreads/library350bpinsert.R2trimmed.fq.gz': discarded 474 reads shorter than 51 bases
`/drive/andre/bird/PEreads/library350bpinsert.R1trimmed.fq.gz': discarded 425 reads shorter than 51 bases
`/drive/andre/bird/PEreads/library3.pe.R1trimmed.fq.gz': discarded 12896079 reads shorter than 51 bases
`/drive/andre/bird/PEreads/library3.pe.R1trimmed.fq.gz': discarded 11809 reads containing non-ACGT characters
`/drive/andre/bird/PEreads/library3.pe.R2trimmed.fq.gz': discarded 12966974 reads shorter than 51 bases
`/drive/andre/bird/PEreads/library3.pe.R2trimmed.fq.gz': discarded 6233 reads containing non-ACGT characters
`/drive/andre/bird/PEreads/library4.pe.R1trimmed.fq.gz': discarded 13926000 reads shorter than 51 bases
`/drive/andre/bird/PEreads/library4.pe.R1trimmed.fq.gz': discarded 7284 reads containing non-ACGT characters
`/drive/andre/bird/PEreads/library4.pe.R2trimmed.fq.gz': discarded 13530720 reads shorter than 51 bases
`/drive/andre/bird/PEreads/library4.pe.R2trimmed.fq.gz': discarded 2405 reads containing non-ACGT characters
`/drive/andre/bird/PEreads/library550bpinsert.R1trimmed.fq.gz': discarded 1541 reads shorter than 51 bases
`/drive/andre/bird/PEreads/library550bpinsert.R2trimmed.fq.gz': discarded 1738 reads shorter than 51 bases
make: *** [birdoutput-1.fa] Error 1
```

**Comment by benvvalk** [comment]
Hi Andre,

Your `abyss-pe` command looks good.  Unfortunately there is no error message in your log to indicate what the problem might be.  Are you sure that your are capturing both STDOUT and STDERR to the log file?

Also, I recommend adding `v=-v` to your `abyss-pe` command and rerunning the assembly.  `v=-v` enables verbose logging and will provide much more information about what the program was doing when it died.

**Comment by aersoares81** [comment]
Hi Ben,
Thank you for letting me know about v=-v, I love verbose/insightful outputs like that.

So, I ran again, the same command, and it kept running this time, to the end! I don't know what caused the error message before, maybe some disk error? Not using nohup? I don't know, I'm running it again now, and will let you know if the error comes up.

Before I close this Issue, I wanted to say that from all the assemblers I've used so far, the ones available on Github or not, this team is the most responsive and friendly one I've ever seen. Thanks for your work, guys! 👍
